### ISSUE TITLE
Overfitting to fine-tuning dataset

### ISSUE THREAD

**Issue by noowad** [google-research/bert#390]
I am finetuning the pre-trained model using my own dataset for next sentence prediction (pos for actual next sentence, neg for fake next sentence)
My dataset consists of about 100,000 examples and I use 80% of the examples for a training set, 10% for a evaluation set, and 10% for a test set.
My issue is that the fine-tuned model shows suspiciously high accuracy for a training set, a evaluation set, and test set, but very poor accuracy (about 50%) for simple examples.

I confirmed that there are no overlapped examples between a training set and a test set, and I trained the pre-trained model only using a training set.

**Comment by noowad** [comment]
@rodgzilla 
The finetuned model made more than 99% accuracy for a test set.
I also confirmed that the performance on some test examples instead of simple examples made high accuracy.
The difference between the test examples and the simple examples is that the length of simple examples is shorter than the length of the test examples. (test examples: about average 100 tokens, simple examples: about average 50 tokens)
Is this a overfitting? 

model running using a test set and simple examples are performed with the same model and the same environment.

**Comment by DeepakSinghRawat** [comment]
@noowad I am facing similar issue. Were you able to solve it?
